src/main/java/org/influxdb/dto/Point.java

```diff
@@ -385,4 +387,51 @@ private StringBuilder formatedTime() {
     return sb;
   }
 
+  public Point tag(final String tagName, final String value) {
+
```

Comment:
Is there a specific tool that will use checkstyle.xml to format the code according to the rules / standards defined within?